05_syntax_lab/04.py

```diff
@@ -1,8 +1,14 @@
-"""
-Write a program that reads lines from the user
-until an empty line is inserted.
-After the user typed in an empty line,
-print all previously inserted lines in reverse
-order (from last to first)
-"""
-
+################## Part 4
+text = []
+while 1:
+    print('Please enter a string')
+    tmp = input()
+    if len(tmp) <= 0:
+        break
+    text.append(tmp)
+    #print(text)
+
+i = len(text)
+while i > 0:
+    print(text[i-1])
+    i -= 1

```

Comment:
Hi,

I am using:

#-*- coding: utf-8 -*-
print(u'\u05d1\u05e8\u05d5\u05db\u05d9\u05dd \u05d4\u05d1\u05d0\u05d9\u05dd')

but get gibberish…
how come?


From: Gabor Szabo [mailto:notifications@github.com]
Sent: Wednesday, July 06, 2016 11:23 AM
To: ynonp/python-examples-verint-2016-07
Cc: Rosenthal, Hanan (Embedded); Author
Subject: Re: [ynonp/python-examples-verint-2016-07] ToCode Solution: תרגול תחביר (#14)


In 05_syntax_lab/04.py<https://github.com/ynonp/python-examples-verint-2016-07/pull/14#discussion_r69691319>:

> -"""

> -

> +################## Part 4

> +text = []

> +while 1:

> +    print('Please enter a string')

> +    tmp = input()

> +    if len(tmp) <= 0:

> +        break

> +    text.append(tmp)

> +    #print(text)

> +

> +i = len(text)

> +while i > 0:

> +    print(text[i-1])

> +    i -= 1

I mean this:
v = input('prompt')

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ynonp/python-examples-verint-2016-07/pull/14/files/527f9d76572f49819db8bac5b07c328a76065521..6a026c693f3f8e4016c08623a548253dec261b0d#r69691319>, or mute the thread<https://github.com/notifications/unsubscribe/ATVHXlxWRNED_h7qo9DM0ulzTgPn8VpAks5qS2X7gaJpZM4JE3pq>.

-- 
This electronic message may contain proprietary and confidential 
information of Verint Systems Inc., its affiliates and/or subsidiaries.
The information is intended to be for the use of the individual(s) or
entity(ies) named above. If you are not the intended recipient (or 
authorized to receive this e-mail for the intended recipient), you may not 
use, copy, disclose or distribute to anyone this message or any information 
contained in this message. If you have received this electronic message in 
error, please notify us by replying to this e-mail.